justfile

```
@@ -256,21 +256,21 @@ bootstrap *ARGS:
 # --------------------------------------------------
 
 @tailwind *ARGS:
-    npx tailwindcss \
-        --config ./static/js/tailwind.config.js \
-        --input ./static/css/tailwindcss.css \
-        --output ./static/css/tailwindcss.min.css \
-        {{ ARGS }}
+    docker compose run django tailwind \
+    -i tailwind.css \
+    -o static/css/tailwind-output.css \
+    {{ ARGS }}
```

Same as what I mentioned in the comments on those files, I would keep what the existing files are in the project. Also, given the command's existing long flags, I would keep those as well.

```suggestion
    docker compose run django tailwind \
        --input ./static/css/tailwindcss.css \
        --output ./static/css/tailwindcss.min.css \
        {{ ARGS }}
```

Though, TBH I'm not familiar enough with the project to get this, but there is a little bit of misdirection here? Looking at those files, the input looks like its minified and the min one looks like it's not? I'll defer to the regular maintainers on this one.

---

Also, the input and output *may* not be needed since this is now using django-tailwind-cli and those are set in the config for that in `DJANGO_SETTINGS_MODULE`. The preference may be to keep this command using the node package, though again I'll defer to the maintainers.

If that's the case, just need to update the npx call to the new name of the cli:

```suggestion
    npx @tailwindcss/cli \
        --config ./static/js/tailwind.config.js \
        --input ./static/css/tailwindcss.css \
        --output ./static/css/tailwindcss.min.css \
        {{ ARGS }}
```